cypress/integration/footerSpec.js

```diff
@@ -26,9 +26,12 @@ describe('Footer Tests', () => {
   it('should contain copyright text', () => {
     cy.get('footer p').should(
       'contain',
```

Comment:
Can we use `have.text` assertion here to make sure that the `copyright` symbol and the text appears in the correct order. On the snapshot i can see we are loosing the `copyright` symbol.

Reply:
@sadickisaac `cy.get('footer p')` a concatenation of all the text in the footer, not just the copyright text. Looking at https://github.com/bbc/simorgh/pull/2847/files#diff-40b843802de7eef375e3df8299b07701, I'm not sure why we need to test if the text appears in the correct order

Reply:
If i have this markup on the footer
``` html
<footer>
  <p> footer test </p>
  <span> copyrightSymbol </span>
</footer>
```
The cypress test would still pass because it is checking for the first occurrence of `<span>`. This should fail because that is not the correct order.